app/api/questions/next-unanswered/route.ts

```
@@ -0,0 +1,74 @@
+import { NextRequest, NextResponse } from 'next/server'
+import { createClient } from '@/lib/supabase/server'
+import { prisma } from '@/lib/db'
+
+export const dynamic = 'force-dynamic'
+
+export async function GET(request: NextRequest) {
```

ESLint

[error] 'request' is defined but never used.

(@typescript-eslint/no-unused-vars)